FreeRTOS/Test/CBMC/proofs/Queue/QueueGenericCreate/QueueGenericCreate_harness.c

```diff
@@ -32,19 +32,14 @@
 #include "cbmc.h"
 
 
-void harness(){
-	UBaseType_t uxQueueLength;
-	UBaseType_t uxItemSize;
-	uint8_t ucQueueType;
+void harness()
+{
+    UBaseType_t uxQueueLength;
+    UBaseType_t uxItemSize;
+    uint8_t ucQueueType;
 
-	size_t uxQueueStorageSize;
-	__CPROVER_assume(uxQueueStorageSize < (UINT32_MAX>>8));
+    /* Allow CBMC to run in a reasonable amount of time. */
+    __CPROVER_assume( ( uxQueueLength == QUEUE_LENGTH ) || ( uxItemSize == QUEUE_ITEM_SIZE ) );
```

Comment:
This assumption seems very restrictive...

If there is no other way, can we at least define the `QUEUE_LENGTH` and `QUEUE_ITEM_SIZE` to various values in the configurations file above? That would broaden the scope of the proofs.

Reply:
It's binding one or the other, but not both at once; i.e., when queue length is fixed, item size can be anything, and when item size is fixed, queue length can be anything.  I used macros with the intent of providing additional values in a future PR.

Reply:
Makes sense. Thanks.